cnf-certification-test/networking/icmp/icmp.go

```diff
@@ -61,17 +61,20 @@ func BuildNetTestContext(pods []*provider.Pod, aIPVersion netcommons.IPVersion,
 				logger.Info("Skipping pod %q because it is excluded from %q connectivity tests only", put.Name, aType)
 				continue
 			}
-			for netKey, multusIPAddress := range put.MultusIPs {
+			for netKey, multusNetworkInterface := range put.MultusNetworkInterfaces {
 				// The first container is used to get the network namespace
-				processContainerIpsPerNet(put.Containers[0], netKey, multusIPAddress, netsUnderTest, aIPVersion, logger)
+				// Loop through all the network interfaces and their IPs
+				for _, networkInterface := range multusNetworkInterface {
+					processContainerIpsPerNet(put.Containers[0], netKey, networkInterface.IPs, networkInterface.Interface, netsUnderTest, aIPVersion, logger)
```

Comment:
Same here, we don't need a double loop just get the list of IPs from the single CniNetworkInterface per Net (see previous comment)

Reply:
Done.